datetime/src/main/scala/com/fortysevendeg/scalacheck/datetime/jdk8/GenJdk8.scala

```diff
@@ -54,4 +53,14 @@ object GenJdk8 extends GenJdk8
 object ArbitraryJdk8 extends GenJdk8 {
   implicit def arbJdk8(implicit granularity: Granularity[ZonedDateTime]): Arbitrary[ZonedDateTime] =
     Arbitrary(genZonedDateTime)
+
+  implicit val arbInstant: Arbitrary[Instant] = {
+
+    implicit val c: Choose[Instant] =
+      Choose.xmap[Long, Instant](Instant.ofEpochSecond(_), _.getEpochSecond)
+
+    Arbitrary(
+      Gen.choose[Instant](Instant.MIN, Instant.MAX)
+    )
+  }
```

Comment:
In order to use the granularity we could write something like:

```scala
implicit def arbInstant(implicit zdtArb: Arbitrary[ZonedDateTime]): Arbitrary[Instant] =
  Arbitrary(zdtArb.arbitrary.map(_.toInstant))
```

Reply:
Is there an `Arbitrary[ZonedDateTime]` already in scope, @fedefernandez, such that we don't need the `implicit zdtArb: Arbitrary[ZonedDateTime]`?

I could be missing something too in your suggestion.

Reply:
hey @kevinmeredith I just found an old branch with this instance and some others (`LocalDateTime` and `LocalDate`). I've cleaned up a bit and create a PR, feel free to take a look

https://github.com/47deg/scalacheck-toolbox/pull/66

Reply:
👍